pkg/controller/master/upgrade/upgrade_controller.go

```diff
@@ -628,6 +623,55 @@ func upgradeEligibilityCheck(upgrade *harvesterv1.Upgrade, repoInfo *RepoInfo) (
 	return true, ""
 }
 
+func (h *upgradeHandler) prepareNodesForUpgrade(upgrade *harvesterv1.Upgrade, repoInfoStr string) (*harvesterv1.Upgrade, error) {
+	upgradeConfig, err := settings.DecodeConfig[settings.UpgradeConfig](settings.UpgradeConfigSet.Get())
+	if err != nil {
+		return upgrade, err
+	}
+	logrus.WithFields(logrus.Fields{
+		"upgrade_namespace": upgrade.Namespace,
```

Comment:
Nit: Although this doesn't look wrong, should we use 'namespace' 'name' 'kind' as defined in https://github.com/harvester/harvester/blob/master/enhancements/20240327-structured-logging.md?

Reply:
Thanks for the heads-up. We should follow what we proposed :)